src/template.py

```diff
@@ -10,13 +10,17 @@ def __init__(self, template_file=None):
         """
         Initializes the Template object.
 
+        Attributes: template_content (dict): Content of the template,
```

Comment:
plaease clean up the formatting (add a newline after "Attributes" and indent the next one!